README.md

```diff
@@ -26,31 +26,51 @@ Images are uploaded from the cloud and may be publicly accessible - they use ran
 
 Example walk through - this assumes you already have an account with access to functions and events,
 
-I suggest setting up an OCI account with only read/write access to the appropriate buckets (the account with private key /Users/OCliffe/.oci/oci_faces_key.pem )
+I suggest setting up an OCI account with only read/write access to the appropriate buckets (the account with private key /Users/username/.oci/oci_faces_key.pem ).
```

Comment:
I don't think you need the private keys any more after RP changes right, remove the brackets thing